go.mod

```diff
@@ -110,8 +111,8 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver v0.54.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zookeeperreceiver v0.54.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.7.5
-	go.opentelemetry.io/collector v0.54.0
+	github.com/stretchr/testify v1.8.0
+	go.opentelemetry.io/collector v0.55.0
```

Comment:
Our distro is currently based on v0.54.0 of the collector, could we change this PR to include version v0.54.0 of the logzio exporter instead? We'll update to v0.55.0 probably sometime later this week, or early next week.

Reply:
@BinaryFissionGames @yotamloe 

I'm ok with merging this at v0.55.0. I'm working on the `v0.55.0` update today. It seems like all CI is currently passing with the mixed version.

Reply:
Sure makes sense, ill change to v0.54.0